src/libdrakvuf/libdrakvuf.h

```diff
@@ -223,6 +223,28 @@ struct drakvuf_trap {
     void *data;
 };
 
+
+////////////////////////////////////////////////////////////////////////////
+
+// IMHO these definitions must be placed within another file, named
+// libdrakvuf-windows.h or something similar
+
+// For get_previous_mode...
+typedef enum _privilege_mode_t {
```

Comment:
The standard style on many projects is this part is simply `privilege_mode` and the typedef'd version is the one followed by _t.